py-polars/tests/unit/functions/range/test_datetime_range.py

```diff
@@ -579,3 +581,34 @@ def test_datetime_range_specifying_ambiguous_11713() -> None:
         "datetime", [datetime(2023, 10, 29, 2), datetime(2023, 10, 29, 3)]
     ).dt.replace_time_zone("Europe/Madrid", ambiguous=pl.Series(["latest", "raise"]))
     assert_series_equal(result, expected)
+
+
+@given(
+    closed=st.sampled_from(["none", "left", "right", "both"]),
+    time_unit=st.sampled_from(["ms", "us", "ns"]),
+    n=st.integers(1, 100),
+    unit=st.sampled_from(["s", "m", "h", "d", "mo"]),
+    start=st.datetimes(datetime(1965, 1, 1), datetime(2100, 1, 1)),
+)
+def test_datetime_range_fast_slow_paths(
```

Comment:
this runs in about 0.71s. well-worth it imo

Reply:
Can we only run this in the benchmark test? That means they run on a perf release. I do think 0.71s is very long ;)